dll/win32/browseui/shellbrowser.cpp

```diff
@@ -315,6 +317,8 @@ class CShellBrowser :
     DWORD m_BrowserSvcFlags;
     bool m_Destroyed;
     BYTE m_NonFullscreenState;
+
+    enum { FSF_MBAR = 0x1, FSF_SBAR = 0x2, FSF_RESIZE = 0x4, FSF_MAXIMIZED = 0x8 };
```

Comment:
```suggestion
    enum { FSF_MBAR = 0x1, FSF_SBAR = 0x2, FSF_RESIZE = 0x4, FSF_MAXIMIZED = 0x8 };

```
(separing the 'public' from the rest).

Question: does the "FSF" prefix stand for "fullscreen flag" ?

Reply:
They should perhaps be named non-fullscreen flags but yes, they are flags used to remember what the "correct" state is once we exist fullscreen.